### PR TITLE
Bugfix/product types

### PR DESCRIPTION
**Pull request by princepatoliya**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

- **What is the current behavior?** (You can also link to an open issue here)

- **What is the new behavior (if this is a feature change)?**

- **Other information**:

### CODE REVIEW

**Review by akartik80 — APPROVED** [review]
Please bump version

File: src/types/Enums.ts [hunk 1]
```diff
@@ -48,3 +48,8 @@ export enum CountryCode {
 export enum DiscountCodeType {
   COUPON,
 }
+
+export enum PlatFormTitle {
+  AMAZON,
+  NYKAA,
```
Comment (akartik80) [inline src/types/Enums.ts:54]:
```suggestion
  NYKAA
```

**Review by akartik80 — APPROVED** [review]

File: src/types/Enums.ts (hunk 1 above)
Comment (princepatoliya) [reply src/types/Enums.ts:54]: ye prettier se aa raha hai

Comment (princepatoliya) [reply src/types/Enums.ts:54]: hamne discuss kiya tha ki abhi jo prettier kar raha hai karne do, jab eslint and prettier sab repo se setup karenge tab ek default structure bana denge sab ke liye

**Pull request merged**